docker-compose.yml

```diff
@@ -1,6 +1,16 @@
 version: '2.2'
 
 services:
+  certs:
+    image: nginx
```

Comment:
Why nginx?

Reply:
It has openssl. I've tried to use [alpine/openssl](https://github.com/alpine-docker/openssl), but I also need the shell to do everything in one line.